mods/default/mapgen.lua

```diff
@@ -113,6 +113,27 @@ function default.register_mgv6_ores()
 		},
 	})
 
+	-- Dry Dirt
+
+	minetest.register_ore({
```

Comment:
Savanna biome doesn't occur in mgv6 so this ore is not needed.

Reply:
done